lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -332,6 +332,25 @@ sub _getSoftwaresList {
             $software->{INSTALLDATE} = _dateFormat(_keyLastWriteDateString($data));
         }
 
+		#----- SQL Server -----
```

Comment:
Please replace each tabulation by 4 spaces to follow our source conventions. Thx.